python/ray/autoscaler/command_runner.py

```diff
@@ -75,8 +75,12 @@ def remote_shell_command_str(self) -> str:
         """Return the command the user can use to open a shell."""
         raise NotImplementedError
 
-    def run_init(self, *, as_head: bool, file_mounts: Dict[str, str],
-                 sync_run_yet: bool) -> Optional[bool]:
+    def run_init(self,
+                 *,
+                 as_head: bool,
+                 file_mounts: Dict[str, str],
+                 sync_run_yet: bool,
+                 with_output: bool = True) -> Optional[bool]:
```

Comment:
Note: it seems like we were already relying on this `with_output` arg, since it's in the `SSHCommandRunner` and was being inherited from there.